index.html

```diff
@@ -206,6 +207,15 @@
         empty string. If the provided value is not a [valid request
         destination], then [queue a task] to [fire a simple event] named
         `error` at the [link] element and abort these steps.[[!FETCH]]</li>
+        <li>If the `type` attribute's value is null or the empty string, then
```

Comment:
An attribute cannot have a null value. Presumably you mean it is not present? Here and for media.